src/model/quotes.js

```diff
@@ -66,13 +66,10 @@ class QuotesModel {
 
     // make sure initiator is the PAYER party
     if (quoteRequest.transactionType.initiator !== 'PAYER') {
-      throw new Errors.FSPIOPError(quoteRequest, 'Only PAYER initiated transactions are supported',
-        fspiopSource, Errors.ApiErrorCodes.SERVER_ERROR)
+      throw ErrorHandler.CreateFSPIOPErrorFromErrorCode(2000, 'Only PAYER initiated transactions are supported', null, fspiopSource)
```

Comment:
why 2000? is there an enum for the error code?

Reply:
Now using NOT_IMPLEMENTED